package.json

```diff
@@ -21,6 +21,7 @@
     "react-router-dom": "^6.18.0",
     "react-scripts": "5.0.1",
     "redux-persist": "^6.0.0",
+    "sweetalert": "^2.1.2",
```

Comment:
Why are you installing it?

Reply:
The app was giving errors before installing sweetalert, once I did it started working.

Reply:
Please revert this changes (it applies the same for package-lock.json) y just run `npm install`, it should fix this issue. Otherwise, please share an screenshot about the issue